### DISCUSSION TITLE
TUYA TS0601 / TZE200 - Problems with humidity accuracy

### DISCUSSION THREAD

**Discussion by muerue** [Koenkk/zigbee2mqtt#22593] [Devices]
Hello - I am using several sensors of this type from multiple vendors.
The remperature value is alwaya fine but the humidity is real strange.
Sensors lying in free air are reporting "0" as expected. Using them in the soil, the delivered value is alway around 80 - 100% or more.
Where can be the problem?

Are the sensors that bad can something be wrong with the fields layout and/or size?

Any suggestion any helping idea qould be fine...

Thanks

RUE

**Comment by Levynger** [comment]
Same issue here..,
